apps/docs/src/@primer/gatsby-theme-doctocat/components/header.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import styles from './header.module.css'
+import {MarkGithubIcon} from '@primer/octicons-react'
+import {Navigation} from '../../../components/navigation/navigation'
+import {SidePanel} from '../../../components/side-panel/side-panel'
+
+export const HEADER_HEIGHT = 56
+
+export default function Header() {
+  return (
+    <header className={styles.PageHeader}>
+      <a href="https://primer.style/" className={styles.SiteTitle}>
```

Comment:
Can we make this link to Primer Brand docs? Our users rarely need to go to the homepage or product docs so it feels intuitive to send them to the root. Ideally it would work like our marketing pages, where clicking on the title text takes you back to Primer Brand UI homepage and clicking the Logo icon will take you back to primer.style. What do you think?

https://github.com/user-attachments/assets/8960d252-667f-4bab-af66-4c11a1659fb5

Reply:
@mperrotti 👋🏽 After a conversation with @tallys, we’d like to send the users to the root, as suggested by @rezrah, and insulate Primer Brand from Primer to reduce dependencies.